examples/knative-builder/main.py

```diff
@@ -41,7 +46,7 @@
                        'tensorflow/mnist/logs/fully_connected_feed/', os.getenv('HOSTNAME', ''))
 MODEL_DIR = os.path.join(LOG_DIR, 'model.ckpt')
 
-@Train(repository='<your-repository>', builder='knative')
+@native.Training()
```

Comment:
Is train() method in the decorated class required or optional now? Is there a way to demonstrate that through examples?

Reply:
Currently it is still required. We can open a follow-up issue to discuss how to remove this requirement.